test/integration/charge_ft_tests.js

```diff
@@ -396,6 +396,25 @@ describe('chargeTests', function () {
           .end(done)
       })
 
+      it('should redirect user from /auth_waiting to the returnURL when connector returns a successful status and is web payment (Apple/Google Pay)', function (done) {
```

Comment:
Nice additional test, plus I see there's already one to confirm it redirects to confirm page for non wallet payments 👍